File: back/src/Chatbot.Model/DataModel/Token.cs
using System;

namespace Chatbot.Model.DataModel
{
    public class Token
    {
        public string TokenId { get; set; }
        public DateTime DateCreated { get; set; }
        public DateTime DateExpired { get; set; }
        public TimeSpan AutoExpired { get; set; }
        public Guid UserId { get; set; }
        public User User { get; set; }
    }
}

File: lk/src/app/contracts/token.ts
export default class token{
    constructor(options: any){
        this.tokenId = options.tokenId;
        this.dateCreated = options.dateCreated;
        this.dateExpired = options.dateExpired;
        this.autoExpired = options.autoExpired;
        this.userId = options.userId;
    }

    tokenId: string;
    dateCreated: Date;
    dateExpired: Date;
    autoExpired: Date;
    userId: string;
}
